impress-org/givewp
# New: Refreshed Donation Forms list UI

Part of #6175

## Description



This PR includes a new Donation Forms admin page which replaces the previous WP table. The new page is built in React and pulls its data from a WP REST API endpoint. This version includes pagination, search, and filtering by post status; additional filters, bulk actions, sorting, and custom columns will be added in later scopes.

## Affects


Any add-ons that extend or modify the forms list won't be compatible with this PR, being included in a later scope.

## Visuals



## Testing Instructions


1. Install GiveWP
2. In the admin, go to Donations -> All Forms
3. View the new Donation Forms page and verify that all of the included functionality works

## Pre-review Checklist
